### ISSUE TITLE
Exporter Needs full path to export correctly

### ISSUE THREAD

**Issue by mamu82** [AsoboStudio/glTF-Blender-IO-MSFS#56]
### Current Behavior

When selecting export folder, 
If the export folder Is the same of the .blend
in the textbox appears "//"
Exporting throws errors in Blender 

### Expected Behavior

When selecting a folder, the full path name must be provided to avoid erros

### Steps To Reproduce

Select as export folder the same folder where the .blend file Is located and click accept

### Environment

```markdown
- OS:win.10 ita
- Blender: 3.0.1
- glTF-Blender-IO-MSFS: 1.0.5
```

**Comment by pepperoni505** [comment]
Hi,
When entering `//` as a texture path, it is considered as the root of your drive, for instance `C:/texture.png`. Because it's being exported at that base level, administrative permissions are needed, which is why it fails. To export at the same level as your blend file, please leave that field blank.

**Comment by mamu82** [comment]
I didnt' put in "//" myself
 the "Folder:" textbox get populated as soon as I choose the folder where I want the gltf to export to by pressing "Accept"
made another try, selecting other folders i obtain more strange results as export path like "//..\..\Downloads\" when selecting windows Download folder in Documents folders
BUT
as soon as I choose a folder on other Hard Drive (D:), the path becomes correct, and I can choose any folder in my documents folder and gets it right path

Reloading blender, same story again

**Comment by pepperoni505** [comment]
I can't recreate this. Did you use the multi-exporter?

**Comment by mamu82** [comment]
yep, multiexporter 2.0

repro video

https://youtu.be/kmtrnD2uqtg

**Comment by pepperoni505** [comment]
I think I know the issue. When you select your export folder, click the gear icon in the top left and there should be a setting that is along the lines of use relative path - please try toggling that

**Comment by mamu82** [comment]
Confirm, unchecking relative path solve the issue

But that setting is not persistent, and relative path checked comes from vanilla installation.  Pretty annoying for average users, and I'm below average :)
I have saved a new startup .blend file with that option uncheck to avoid forgetting toggling the check box, 
if it is not possible to fix in the the plugin/addon itself, this would deserve a note in installation readme

(if you need me to close the issue, will do, i had already find the workaround even before opening it, but will try to have a perfect plugin!)

**Comment by pepperoni505** [comment]
Opened a pull request that fixes this issue: #58
